src/subscription_manager/repolib.py

```diff
@@ -167,6 +168,11 @@ def enable_pkg_plugins(cls):
             log.debug('The rhsm.auto_enable_yum_plugins is disabled. Skipping the enablement of yum plugins.')
             return []
 
+        dist_info = HardwareCollector().get_distribution()
+
+        if dist_info[4].lower() == "debian" or "debian" in dist_info[5]:
```

Comment:
no need to lowercase the distribution ID

Reply:
Missed that one. => Fixed